pkg/features/feature.go

```diff
@@ -87,6 +100,15 @@ func InitializeFeatures(ctx *config.ScaledContext, featureArgs string) {
 				}
 			}
 
+			// checks if developer has changed description value from previous rancher version
+			if featureState.Status.Description != f.description {
+				newFeatureState := featureState.DeepCopy()
+				newFeatureState.Status.Description = f.description
+				if featureState, err = ctx.Management.Features("").Update(newFeatureState); err != nil {
+					logrus.Errorf(errMsg, f.name, err)
```

Comment:
Should either return nil here or not assign to same variable as it could be mangled/empty coming back, and then the function would continue with that.

Reply:
good catch